packages/server/src/index.ts

```diff
@@ -36,29 +39,45 @@ const runServer = (
     didResolver,
   })
 
+  const mailTransport =
+    config.emailSmtpUrl !== undefined
+      ? createTransport(config.emailSmtpUrl)
+      : createTransport({ jsonTransport: true })
+
+  const mailer = new ServerMailer(mailTransport, config)
+
   const app = express()
   app.use(cors())
   app.use(loggerMiddleware)
 
+  const locals: Locals = {
```

Comment:
good call on pulling this out from the middleware 👍 

since we overwrite the logger in the middleware (since that one _is_ request specific), can we not set it here?

`Omit<Locals, 'logger'>`

Reply:
Can do!  Is the idea to omit `logger` from `app.locals` entirely, or just to change how we piece these types together for `app.locals` versus `res.locals`?

Reply:
ohh i see the point of that now

okay never mind, leave this as is, but maybe add a quick comment next to the middleware part (`logger: req.log`) noting that the logger is request specific.